src/HackF5.UnitySpy/Detail/MemoryObject.cs

```diff
@@ -1,31 +1,40 @@
-﻿namespace HackF5.UnitySpy.Detail
+﻿using System;
+
+namespace HackF5.UnitySpy.Detail
 {
     /// <summary>
     /// The base type for all objects accessed in a process' memory. Every object has an address in memory
     /// and all information about that object is accessed via an offset from that address.
     /// </summary>
     public abstract class MemoryObject : IMemoryObject
     {
-        protected MemoryObject(AssemblyImage image, uint address)
+        protected MemoryObject(AssemblyImage image, IntPtr address)
         {
             this.Image = image;
             this.Address = address;
         }
 
+        public IntPtr GetAddress()
```

Comment:
I'm not really fluent in C#, so please forgive me if this is a dump question, but why have a `GetAddress()` method in addition to the `Address` getter?